tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -205,9 +203,9 @@ jobs:
     inputs:
       ArtifactName: onnxruntime
 
-  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
-    displayName: 'Component Detection'
-    condition: succeeded()
+    - template: component-governance-component-detection-steps.yml
+      parameters :
+        condition : 'succeeded'
```

Comment:
miss ()

Reply:
template parameter is compile time string

---
In reply to: [367708208](https://github.com/microsoft/onnxruntime/pull/2857#discussion_r367708208) [](ancestors = 367708208)